zuul.d/projects.yaml

```diff
@@ -75,6 +75,12 @@
       - integrated-queue
       - publish-to-galaxy
 
+- project:
```

Comment:
This needs to be a 2nd PR. We first need to load the project into zuul, then we can assign jobs.